packages/utils/src/getCSVSeparator.ts

```diff
@@ -0,0 +1,30 @@
+export const LOCALES_USING_SEMICOLON = [
```

Comment:
I would prefer either:

1. Create a typesafe map to map the langages from `Locale` type onto a `bool` explicitly defining  the usage of semicolon/comma. When we add a new language Typesscript will force the developer to add it here.

2. Using some tricky-trick like `(new Intl.NumberFormat(locale).format(1.1)).includes(',')` to determine if language uses comma as separator

Reply:
Something like this: 
```suggestion
export const LOCALES_USING_SEMICOLON: Record<Language, ";"|","> = [
```